src/settings/Input.hpp

```diff
@@ -3,6 +3,7 @@
 #include <SFML/Graphics.hpp>
 #include <string>
```

Comment:
Unused includes, although std::vector is in use in this class.

```suggestion
#include <vector>
```